components/AlumniCompanies/alumni-companies.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from 'react';
+import Image from 'next/image';
+import { SwitchTransition, CSSTransition } from 'react-transition-group';
+
+import { urlForImage } from '@/lib/sanity';
+import { onUpdateCompaniesList } from './utils';
+import { AlumniCompaniesP, ArrayAlumniCompanies } from '../../types';
+
+import styles from './styles.module.scss';
+
+const AlumniCompanies: React.FC<AlumniCompaniesP> = props => {
+  const { companies } = props || {};
+
+  const [companiesList, setCompaniesList] = useState<ArrayAlumniCompanies[]>(
+    companies.slice(0, 6)
```

Comment:
```suggestion
    companies.slice(0, companies.length)
```
why 6 ?)

Reply:
Maybe create limit variables?

Reply:
This is the number of items displayed. Corrected in the new committee